README.rst

```diff
@@ -0,0 +1,376 @@
+.. isbg documentation master file, initially created by
+   pandoc --from=markdown_github --to=rst --output=README.rst README.md
+
+
+IMAP Spam Begone
+================
+
+**isbg** is a script and a python module that makes it easy to scan an
+IMAP inbox for spam using *SpamAssassin* and get your spam moved to another
+folder.
+
+Unlike the normal mode of deployments for *SpamAssassin*, isbg does not need to
+be involved in mail delivery, and can run on completely different machines to
+where your mailbox actually is. So this is the perfect tool to take good care
+of your ISP mailbox without having to leave it.
+
+You can read the full documentation in `Read the docs`_.
+
+.. _Read the docs: https://isbg.readthedocs.io/
+
+.. only:: html
+
+    .. contents:: Table of Contents
+       :depth: 3
+
+Features
+--------
+
+-  Works with all common IMAP servers
+-  Works on Linux, MacOS X and Windows (even smartphones!)
+-  Can do IMAP over SSL
+-  Can remember your password
+-  Will work painlessly against multiple IMAP accounts and servers
+-  Is not involved in the mail delivery process, and so can run on any
+   machine
+   that can contact your IMAP server
+-  Highly configurable
+-  Sensible defaults so you don't have to do any configuring :-)
+-  Compatibility with Python 2.7, 3.5, 3.6
+-  Possibility to skip spam detection to stick only to the teach feature
+-  Don't fail when meeting horrible and bad formed mail
+-  Lock file to prevent multiple instance to run at the same time (for
+   cron jobs)
+
+
+Installation
+------------
+
+**isbg** install a python package module and also a script to use it ``isbg``,
+it also install another script to unwrap messages: ``isbg_sa_unwrap``.
+
+Dependencies
+~~~~~~~~~~~~
+
+**isbg** is written in the Python language. Python is installed by default on
+most Linux systems. You can can find out more about Python at
+`Python home page`_.
+
+Make sure you have *SpamAssassin* installed. All the necessary information
+can be found on the `SpamAssassin wiki`_. *SpamAssassin* should be on your
+``$PATH`` (it installs in ``/usr/bin/`` by default)
+
+To run, ``isbg`` also depends on some python modules.
+
+- `docopt`_ for command line options.
+
+- `cchardet`_ or `chardet`_  for encoding detection.
+
+- `xdg`_ to found the ``.cache`` directory. ``xdg`` is not required, if it's
```

Comment:
I understand `isbg` will still work without `xdg`, but I guess we want people to use it, right? As such, shouldn't it be added to the `requirements.txt` file?